src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -240,22 +243,20 @@
                description [MarkdownView {:text description}]
                :else [:span {:style {:fontStyle "italic"}} "No description provided"])))
      (when-not (empty? library-attributes)
-       [LibraryView {:library-attributes library-attributes
-                     :library-schema library-schema
-                     :workspace workspace
-                     :workspace-id workspace-id
-                     :request-refresh request-refresh
-                     :can-share? can-share?
-                     :owner? owner?
-                     :curator? curator?
-                     :writer? writer?}])
-     [attributes/WorkspaceAttributeViewerEditor {:ref "workspace-attribute-editor"
-                                                 :editing? editing?
-                                                 :writer? writer?
-                                                 :workspace-attributes workspace-attributes
-                                                 :workspace-bucket bucketName
-                                                 :workspace-id workspace-id
-                                                 :request-refresh request-refresh}]]))
+       [LibraryView (utils/restructure
+                     library-attributes
+                     library-schema
+                     workspace
+                     workspace-id
+                     request-refresh
+                     can-share?
+                     owner?
+                     curator?
+                     writer?
+                     catalog-with-read?)])
```

Comment:
Let's adjust formatting so we don't use a ton of lines for this.  Same in the other place.